bip-0341.mediawiki

```diff
@@ -284,6 +284,111 @@ The reason for this is to increase leaf entropy and prevent an observer from lea
 
 == Test vectors ==
 
+=== SIGHASH ALL ===
+  
+  The following is an unsigned transaction:
+    02000000025f6092ec9bb430830dfc344260dd5a03cf355186e774be49b2fe5c362f56cb8d00000000000000000061431892d76aa28b5ed1e3da8800fa0d7190c4b4f22be5f416d2d07e573b32e10100000000000000000100ca9a3b000000001976a914682dfdbc97ab5c31300f36d3c12c6fd854b1b35a88ac00000000
+  
+  The first input comes from a P2WPKH
+    scriptPubKey : 0014196a5bea745288a7f947993c28e3a0f2108d2e0a value: 5.0
+    private key  : 6b3973ee2ce444ada0147716925f6f77569350804835498593dd3be95163d558
+    public key   : 0271be339aeae9ed2c6a5a7f8ac5f49638da387612be881c7ed2fb3848b0ef8a6c
+    
+  The second input comes from a P2TR
+    scriptPubKey : 512029d942d0408906b359397b6f87c5145814a9aefc8c396dd05efa8b5b73576bf2, value: 6
+    private key  : cf3780a32ef3b2d70366f0124ee40195a251044e82a13146106be75ee049ac02
```

Comment:
It looks like this is the tweaked private key corresponding to the listed `scriptPubKey` (or alternatively, the scriptPubKey isn't tweaked at all). I would strongly prefer to only include test vectors that actually follow the procedure listed in the BIP for constructing taproot scriptPubKeys.

Reply:
Right I forgot to tweak the keys. I'll update the pull request with the necessary modifications

Reply:
LGTM. If bitcoin core passes the tests I think they should be merged as soon as possible